websites/O/OG Network/presence.ts

```diff
@@ -0,0 +1,77 @@
+const presence = new Presence({ clientId: "1237999667023708231" }),
+	assets = {
+		logo: "https://www.og-network.net/assets/logo.png",
+	};
+
+let startTime: number;
```

Comment:
no reason to use this after my changes.

Reply:
`startTime` is only assigned, not really used. It can be removed.